resources/assets/js/routes.js

```diff
@@ -84,7 +84,7 @@ const routes = [
 		name: 'page'
 	},
 	{
-		path: '/site/:siteId/profile/:profileId',
+		path: '/site/:site_id/profile/:profileId',
```

Comment:
Eek! This breaks a lot of the Vue stuff for profiles.
I recommend instead changing the [media upload component](https://github.com/unikent/astro/blob/develop/resources/assets/js/components/MediaUpload.vue#L22) to use `Number(this.$route.params.siteId || this.$route.params.site_id)` for now. Until we make all routes use the newer format, which matches best practice for prop names (since that's what they effectively become when the props option for routes is set to true).

Reply:
yup we noticed that at the end of the day too...

I'd rather make the media upload component receive the siteId via a prop though rather than rely on the routing.

Reply:
Agreed, ideally we shouldn't rely on routing at all. We can try and move towards that as time goes on.